_config.yml

```diff
@@ -5,6 +5,7 @@ google_analytics: UA-79528055-1
 core_pages_list:       
   "Home": '/'
   "Semantic Specification": '/spec'
+  "Data Semantics": '/data-semantics'
```

Comment:
How about `"Conventions and Standard Names"`? (if it fits)

We could rename the `data-semantics` file and title if you agree.

Sorry to split hairs, but "Data Semantics" is so vague...